command/main.go

```diff
@@ -23,6 +23,10 @@ func Run(args []string) int {
 			args = []string{"version"}
 			break
 		}
+
+		if arg == "-no-color" {
+			os.Setenv("VAULT_OUTPUT_NO_COLOR", "true")
```

Comment:
I think VAULT_CLI_NO_COLOR would be better here

Reply:
Going to remove this flag entirely. If we do the os.Getenv logic check on command.go's `init()` func as discussed, the envvar will be read before this gets applied.

Optionally, we could keep this if we change `init()` to a func that needs to be called explicitly (and call it after the args have been parsed) in main.go's `Run()`.